facia-tool/public/js/widgets/trail.html

```diff
@@ -6,7 +6,8 @@
         underDrag: state.underDrag() && !state.underControlDrag(),
         underControlDrag: state.underDrag() && state.underControlDrag(),
         imageHidden: meta.imageHide,
-        slimEditor: slimEditor
+        slimEditor: slimEditor,
+        visited_article: visited()
```

Comment:
for consistency this should be called `visitedArticle` and you don't need to call the function

`visitedArticle: visited` should do the job, knockout knows how to handle it